src/components/common/NavigatorComponent.js

```diff
@@ -53,4 +56,7 @@ export default class NavigatorComponent extends Component {
   onWillCommitPreview() {
   }
 
+  onBottomTabReselected() {
```

Comment:
Also not used, should we remove it?

Reply:
It's not a common way in js an RN, but it's the best way I found to share navigation functionality. It's a point for overriding in components which extends NavigatorComponent. This way we don't need to place checks like `this.onBottomTabReselected && this.onBottomTabReselected()` when we call it.